js/index.js

```diff
@@ -31,9 +17,67 @@ for (let i = 0; i<skills.length; i++) {
 }
 
 const technicalSkills = document.createElement('h3');
-      technicalSkills.className = 'technical-skills'
-      technicalSkills.innerHTML = "Technical"
+      technicalSkills.className = 'technical-skills';
+      technicalSkills.innerHTML = "Technical";
 
 const skillsHeader = document.querySelector("#skills-header");
 
-skillsHeader.appendChild(technicalSkills);
\ No newline at end of file
+skillsHeader?.appendChild(technicalSkills);
+
+///MESSAGES///
+
+const messageForm = document.querySelector('form');
+const messages = document.querySelector('#messages');
+
+messageForm?.addEventListener('submit', submission => {
+      submission.preventDefault();
+      messages.style.display = "block";
+      const usersName = submission.target?.usersName.value;
+      const usersEmail = submission.target?.usersEmail.value;
+      const usersMessage = submission.target?.usersMessage.value;
+            console.log(usersName);
+            console.log(usersEmail);
+            console.log(usersMessage);
+      const messageList = messages?.querySelector('ul');
+      const newMessage = document.createElement('li');
+            newMessage.innerHTML =`<a href="mailto:${usersEmail}" target="_blank" rel="noopener noreferrer"> ${usersName} </a><span>${usersMessage}</span>`;
+            const removeButton = document.createElement('button');
+                  removeButton.className = 'removeButton';
+                  removeButton.innerHTML = 'Remove';
+                  newMessage.appendChild(removeButton);
+                  removeButton.addEventListener('click', removeEntry => {
+                        const entry = removeButton.parentNode;
+                        removeEntry = entry?.remove();
+                  });
+            // const editButton = document.createElement('button');
```

Comment:
Probably a good idea to add a TODO above the commented out code. Helps with keeping track of what needs to get done.